src/main/java/com/gradle/ccud/adapters/reflection/Java8MethodHandleLookup.java

```diff
@@ -0,0 +1,33 @@
+package com.gradle.ccud.adapters.reflection;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+class Java8MethodHandleLookup implements MethodHandleLookup {
```

Comment:
💬 Shameless copy of the utility classes available in the Develocity plugin. Ideas how to share these classes are welcome.